src/utils/setupWatcher.js

```diff
@@ -86,9 +91,57 @@ async function itemListedCallback(
     cards
   }
 
+  console.log("item-listed-scraper-event")
   PubSub.publish(EVENT_TYPES.ItemListed, itemListedEvent);
 }
 
+async function settledCallback(
+  eventLogs,
+  collectionName,
+  zoombiesContract
+) {
+  /*
+  event Settled(
+  uint256 indexed itemNumber,
+  address nftToken,
+  address saleToken,
+  uint256 bidAmount,
+  address indexed winner,
+  address indexed seller,
+  address royaltyReceiver,
+  uint256 royaltyAmount,
+  uint256[] tokenIds
+);
+   */
+  const { args } = marketInterface.parseLog(eventLogs)
+  const itemNumber = args.itemNumber.toNumber()
+  const tokenIds = args.tokenIds.map((tokenId) => {
+    return tokenId.toNumber()
+  });
+
+  const bidAmount = Number(ethers.utils.formatEther(args.bidAmount));
+  const royaltyAmount = Number(ethers.utils.formatEther(args.royaltyAmount));
+  const cards = await Promise.all(
```

Comment:
Do we need card info when showing settled event in the live feed slide? Each get card is an API call and if we don't need that information we shouldn't make needless calls